content/tutorials/presence.textile

```diff
@@ -31,49 +34,118 @@ Presence enables clients to be aware of other clients that are currently “pres
 
 When someone enters or leaves a channel, or updates their member data, a presence event is emitted to all presence subscribers on that channel. Subscribing to presence events makes it incredibly easy to build an app that shows, in real time, any changes to clients connected to Ably and present on a channel.
 
-The optional payload can be a string, JSON object, JSON array, or binary data, same as with message data.
+The optional payload can be a string, JSON object, JSON array, or binary data: the same as with message data.
+
+Using the "Presence API":/realtime/presence is straightforward. Follow the steps in this tutorial to learn how.
 
-"Using our Presence API":/realtime/presence is easy. Let's get started.
 
 <%= partial partial_version('tutorials/_step-1-create-app-and-api-key') %>
 
 h2.
   default: Step 2 - Install Ably
 
 blang[javascript].
-  To start using Ably in your web app, you first need to include the Ably library. We recommend that you include the latest client library from our CDN using a simple @<script>@ tag. The client library must be instantiated with the API key you copied in Step 1. As we'll be using presence, we also include the @clientId@, which is what we'll appear as in the presence set. Note that in production we recommend you always use the "token authentication scheme":/core-features/authentication#token-authentication for browser clients, however in this example we use an API key for simplicity.
+  Create a blank HTML page called @example.html@, to host your web app:
+
+  ```[html]
+      <!DOCTYPE html>
+      <html lang="en">
+        <head>
+          <meta charset="UTF-8" />
+          <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+          <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+          <title>Ably Presence</title>
+        </head>
+        <body>
+        </body>
+      </html>
+  ```
 
-  Include the code below just before your closing your HTML @</html>@ tag.
+blang[javascript].
+  To start using Ably in your web app, you first need to include the Ably client library. In this example, we recommend that you include the latest client library from our CDN using a simple @<script>@ tag. You should then instantiate the client library with the API key you copied in Step 1.
```

Comment:
Might be worth turning step 1 into a link back to step 1, small benefit that'll be rarely used, but doesn't detract having it.

Reply:
Thanks ... Going to leave these because they're clearly signposted in the left hand menu and links require maintenance :)